specification/compute/resource-manager/Microsoft.Compute/DiagnosticRP/preview/2025-02-01-preview/diagnostic.json

```diff
@@ -979,7 +979,29 @@
           ],
           "x-ms-enum": {
             "name": "RestrictionsFilterType",
-            "modelAsString": false
+            "modelAsString": false,
+            "values": [
+              {
+                "name": "QuotaAndOfferRestrictions",
```

Comment:
I also advise updating the api version given these changes.

Reply:
It's not breaking changes as this API not in use currently.